files/nxlog_exchange_2010.conf

```
@@ -0,0 +1,154 @@
+</Extension>
+
+ <Extension csv_parser_connectivity>
+     Module      xm_csv
+     Fields      date-time, session, source, Destination, direction, description
+ </Extension>
+
+<Extension csv_parser_smtp>
+   Module      xm_csv
+   Fields      date-time, connector-id, session-id, sequence-number, local-endpoint, remote-endpoint, event, data, context
+</Extension>
+
+
+<Input in_app>
+    Module im_msvistalog
+    Query <QueryList><Query Id="0"><Select Path="Application">*</Select></Query></QueryList>
+    Exec if $Message == '' drop();
+    Exec $Hostname = hostname();
+    Exec if $Message =~ s/^\s+//g log_debug("leading whitespace removed");
+</Input>
+<Input in_sys>
+    Module im_msvistalog
+    Query <QueryList><Query Id="0"><Select Path="System">*</Select></Query></QueryList>
+    Exec if $Message == '' drop();
+    Exec $Hostname = hostname();
+    Exec if $Message =~ s/^\s+//g log_debug("leading whitespace removed");
+</Input>
+<Input in_sec>
+    Module im_msvistalog
+    Query <QueryList><Query Id="0"><Select Path="Security">*</Select></Query></QueryList>
+    Exec if $Message == '' drop();
+    Exec $Hostname = hostname();
+    Exec if $Message =~ s/^\s+//g log_debug("leading whitespace removed");
+</Input>
+
+
+<Input messagetracking>
+    Module      im_file
+    File        '%EXCHROOT%\TransportRoles\Logs\MessageTracking\MSGTRK*.LOG'
+    <Exec>
+        if $raw_event =~ /^(\xEF\xBB\xBF)?(date-time,|#)/ drop();
+        else
+        {
+            csv_parser_messagetracking->parse_csv();
+            $EventTime = parsedate($date + " " + $time);
+        }
+    </Exec>
+</Input>
+
+<Input connectivity>
+   Module      im_file
+   File        '%EXCHROOT%\TransportRoles\Logs\Connectivity\CONNECTLOG*.LOG'
+   <Exec>
+      if $raw_event =~ /^(\xEF\xBB\xBF)?(date-time,|#)/ drop();
+      else
+      {
+            csv_parser_connectivity->parse_csv();
+            $EventTime = parsedate($date + " " + $time);
+      }
+   </Exec>
+</Input>
+
+
+<Input smtp_receive>
+   Module  im_file
+   File    '%EXCHROOT%\TransportRoles\Logs\ProtocolLog\SmtpReceive\RECV*.LOG'
+   <Exec>
+      if $raw_event =~ /^(\xEF\xBB\xBF)?(date-time,|#)/ drop();
+      else
+      {
+            csv_parser_smtp->parse_csv();
+            $EventTime = parsedate($date + " " + $time);
+      }
+   </Exec>
+</Input>
+
+<Input smtp_send>
+   Module      im_file
+   File        '%EXCHROOT%\TransportRoles\Logs\ProtocolLog\SmtpSend\SEND*.LOG'
+   <Exec>
+      if $raw_event =~ /^(\xEF\xBB\xBF)?(date-time,|#)/ drop();
+      else
+      {
+         csv_parser_smtp->parse_csv();
+         $EventTime = parsedate($date + " " + $time);
+      }
+   </Exec>
+</Input>
+
+
+
+<Output out>
+   Module om_udp
+   Host graylog.uniqconsulting.local
+   Port 2016
+   OutputType GELF
+</Output>
+
+
+<Route 1>
+   Path        in_app => out
+</Route>
+ 
+<Route 2>
+   Path        in_sec => out
+</Route>
+ 
+<Route 3>
+   Path        in_sys => out
+</Route>
+  
+ 
+<Route 2>
```

Is there a reason to reuse route names?
May be it should be <Route 4>
```suggestion
<Route 4>
```